netdata.spec.in

```diff
@@ -855,16 +901,16 @@ fi
 %package plugin-perf
 Summary: The perf metrics collector for the Netdata Agent
 Group: Applications/System
-Requires: netdata = %{version}
-Conflicts: netdata < %{version}
+Requires: %{name} = %{version}
+Conflicts: %{name} < %{version}
 
 %description plugin-perf
  This plugin allows the Netdata to collect metrics from the Linux perf subsystem.
 
 %pre plugin-perf
 
-if ! getent group netdata > /dev/null; then
-  groupadd --system netdata
+if ! getent group %{name} > /dev/null; then
+  groupadd --system %{name}
```

Comment:
This will work at the moment. But I think we need to introduce long term the netdata_user and netdata_group vars

Reply:
I suggest to totally disable this "old school" `groupadd` and strongly use [systemd-sysusers](https://www.freedesktop.org/software/systemd/man/sysusers.d.html#) for all user/groups tasks.

Why: when `netdata` installed, for example before ceph or docker or nginx, etc package
See: https://github.com/netdata/netdata/blob/master/netdata.spec.in#L461-L466
Results: netdata modules for this services will not work, because if group is not present, netdata user will not be added to this group

We a cover this locally with rules like this:

```shell
[root@ceph-mon0 /]# cat /etc/sysusers.d/99-netdata.conf
m netdata ceph - - -
```

Benefit: package maintainers can use tmpfiles,sysusers shipped with tarball, [rather than doing them manually](https://gitlab.archlinux.org/archlinux/packaging/packages/netdata/-/blob/main/netdata.sysusers?ref_type=heads). When package get more quality and less errors - netdata, as product, performed much better for end-user

Reply:
> I suggest to totally disable this "old school" groupadd and strongly use [systemd-sysusers](https://www.freedesktop.org/software/systemd/man/sysusers.d.html#) for all user/groups tasks.

I had actually started to look into this about two weeks ago before I got sidetracked (again) by other projects. It’s something that we do, indeed, almost certainly want to do in cases when we can reliably determine that we’re on a systemd-based system (and the native RPM packages definitely fit that description).